docs/usage/user-stories/swissquote.md

```diff
@@ -197,18 +199,42 @@ Here is the dashboard for our current scheduler:
 
 <figure markdown>
   ![Swissquote scheduler dashboard](../assets/images/swissquote_stats.png){ loading=lazy }
-  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, July 2022.</figcaption>
+  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, November 2023.</figcaption>
 </figure>
 
 We don’t force any team to use Renovate, each team can decide to opt-in and do it for each project separately.
 
 Some statistics:
 
-- 824 repositories enabled out of about 2000 active repositories
-- 8000 PRs were merged since we installed Renovate
+- 857 repositories enabled out of about 2000 active repositories
+- 11000 PRs were merged since we installed Renovate
 - 239 PRs were merged last month
 - 2 SSDs died on our Renovate machine with the number of projects to clone again and again
 
+### How does the scheduler work ?
+
+The scheduler is a Node.js application which handles an in-memory queue and starts Docker containers to run Renovate on.
+Our custom scheduler application regularly sends points to an InfluxDB which we then display in Grafana.
+
+Here is how it works:
+
+<figure markdown>
+  ![Swissquote scheduler diagram](../assets/images/swissquote_stats_collection.png){ loading=lazy }
```

Comment:
First off: I like the new image! :heart: 

Is _all_ the information from the diagram also explained as plain text below?

Reply:
Thanks :)

Everything is mentioned, but I did not go into detail on purpose and link our conversation in November as a link for more information. Since the rest of the article gives an overview, I didn't want this part to have too many details.

Reply:
That's okay! 😄 Can you, or a maintainer resolve this conversation? I can't resolve my own conversation. 🙃